README.md

```
@@ -1,49 +1,99 @@
 
-`Dengjen Tashkeel` provides a standalone executable called **dengjen-tashkeel** for diacritizing text from the command line.
+**CLI:**
 
 ```bash
-$ dengjen-tashkeel --help
-Arabic-text diacritic restoration using neural networks
+echo "بسم الله الرحمن الرحيم" > input.txt
+dengjen-tashkeel -f input.txt
+```
 
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)